src/modules/refresh.js

```diff
@@ -0,0 +1,13 @@
+import removeAllChildElements from './remove_dom.js';
+import { retrieveScore } from './api_score.js';
+import displayPlayer from './display.js';
+
+export default function refresh(url) {
+  const uList = document.querySelector('ul');
+  removeAllChildElements(uList);
```

Comment:
- [ ] Kindly use the arrow function instead of regular functions. [why you should arrow functions.](https://betterprogramming.pub/why-javascript-arrow-functions-are-useful-aa023bad403a)